collectors/apps.plugin/apps_plugin.c

```diff
@@ -3238,6 +3249,11 @@ void send_resource_usage_to_netdata(usec_t dt) {
                     , update_every
                     , RATES_DETAIL
             );
+        fprintf(stdout,
+                "CHART apps.files '' 'Apps Open Files' 'open files' disk apps.files stacked 140004 %1$d\n"
+                "VARIABLE fdperc = usedfdpercentage '' absolute 1 1\n"
+                , update_every
+        );
```

Comment:
There is a [better place](https://github.com/netdata/netdata/blob/master/collectors/apps.plugin/apps_plugin.c#L3800-L3805) to define the variable. Just check for the chart type to avoid multiple definitions.